Bio/SeqIO/AbiIO.py

```diff
@@ -412,9 +418,9 @@ def iterate(self, handle):
         # fsa check
         is_fsa_file = all(tn not in raw for tn in ("PBAS1", "PBAS2"))
 
-        if is_fsa_file:
+        if is_fsa_file is True:
```

Comment:
I noticed this PR also changes the way we do boolean comparison (I am highlighting this particular line as an example).

Is there a reason we do `is True` and `is False` directly ([when PEP-8 discourages against this style](https://peps.python.org/pep-0008/))?

Reply:
I'd revert that myself.

Reply:
I was under the impression that `... is True` is better because then the variable does not need to be evaluated. But if PEP8 discourages that, I will revert it.

Reply:
Thanks both :slightly_smiling_face: ~ it's a rule of thumb so I suppose there are valid reasons to do that. Still, for these current uses, it's good to be aligned with PEP-8.